src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -116,6 +122,14 @@ object ExtractionPipeline {
       linkedField = "dataset"
     )
 
+    // don't need to use files apart from storing them, so we don't assign an output here
+    extractLinkedEntities(
+      entityToExtract = EncodeEntity.File,
+      matchingField = "@id",
+      linkingEntities = fc_experiments,
+      linkedField = "dataset"
+    )
```

Comment:
I can't tell for sure from the code that's in-view, but I don't think this is safe as-written. Both this call and the existing one above will try to write their outputs under the same prefix. I think it's possible they'll step on each other.

I would `SCollection.unionAll(experiments, fc_experiments)` to get the combination of both streams, and feed that into this `extractLinkedEntities`

Reply:
Okay, that makes sense - I was wondering if I could just list both experiments and fcExperiments but was worried that wouldn't work since extractLinkedEntities is expecting a SCollection[JsonObject] for the linkingEntities parameter. Thank you!